.github/workflows/sync_github_issues_to_azdo.yml

```diff
@@ -0,0 +1,38 @@
+name: Sync issue to Azure DevOps work item
+
+on:
+  issues:
+    # Omit "labeled" and "unlabeled" to work around https://github.com/danhellem/github-actions-issue-to-work-item/issues/70
+    types:
+      [opened, edited, deleted, closed, reopened, assigned]
+  issue_comment:
+    types: [created, edited, deleted]
+
+jobs:
+  alert:
+    if: ${{ !github.event.issue.pull_request && github.event.issue.title != 'Dependency Dashboard' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Choose work item type
+        id: choose_work_item_type
+        run: |
+          if [ "${{ contains(github.event.issue.labels.*.name, 'enhancement') || contains(github.event.issue.labels.*.name, 'user story') }}" == "true" ]; then
+            echo "work_item_type=User Story" >> $GITHUB_OUTPUT
```

Comment:
FYI @zhindes @maxxboehme @hongloonwong When someone files an "enhancement", this creates an unparented user stories. I realize this is not ideal, because someone has to set its parent, but this hasn't been a major problem in other repos.